src/js/media-container.ts

```diff
@@ -188,18 +188,24 @@ template.innerHTML = /*html*/ `
       /* ::slotted([slot=poster]) doesn't work for slot fallback content so hide parent slot instead */ ''
     }
     :host(:not([${Attributes.AUDIO}])[${
-  MediaUIAttributes.MEDIA_HAS_PLAYED
-}]) slot[name=poster] {
+      MediaUIAttributes.MEDIA_HAS_PLAYED
+    }]) slot[name=poster] {
       display: none;
     }
 
-    ::slotted([role="menu"]) {
-      align-self: end;
+    slot[name=dialog]::slotted(*) {
+      z-index: 2;
```

Comment:
in media-chrome-range the input[type=range] element is used as a hover zone which is 14px higher than the actual media-chrome-range. helps with dragging on mobile. it's primarily for layouts where the timeline goes right above the buttons and then it's better that the timeline slightly overlaps the buttons.

I can see the concerns though, we already had a report from Phil on this in the Mux dashboard with Mux players dialog being on `z-index: 100`.

I'll see if there is a workaround.

Reply:
I did some testing on Mux player with the z-indexes removed and that worked fine because we set them in the themes. maybe that's right approach. leave it up to the consumer of the MC elements? 
I removed all the z-indexes in the last commit.